tests/robottelo/test_decorators.py

```diff
@@ -106,7 +106,8 @@ class MockBug(object):  # pylint:disable=R0903
             whiteboard = None
         dec_settings.upstream = False
         decorators._get_bugzilla_bug = lambda bug_id: MockBug()
-        for MockBug.status in BZ_OPEN_STATUSES + BZ_CLOSED_STATUSES:
+        # skip 'CLOSED' state, as it overides whiteboard 'verified in upstream'
```

Comment:
+1 - good to have 

* a test to ensure CLOSED is the last item
* verify if the function returns False when CLOSED status is passed